src/cideMOD/pxD.py

```diff
@@ -276,6 +276,9 @@ def setup(self, mesh_engine=None):
             self.mesh(mesh_engine)
         timer = Timer('Problem Setup')
 
+        #set_log_level(LogLevel.WARNING)
+        set_log_level(LogLevel.ERROR)
+
```

Comment:
I would add this right after the imports:

```suggestion
import logging
set_log_active(True)
set_log_level(logging.ERROR)
logging.getLogger("FFC").setLevel(logging.ERROR)
logging.getLogger("UFL").setLevel(logging.ERROR)
```